.github/workflows/app.yml

```diff
@@ -0,0 +1,36 @@
+name: Test and build Jellyfin Android
+on:
+  push:
+  release:
+    types: [ published ]
```

Comment:
*Okay now in comment form*

This indeed will trigger on all pushes on all branches, yet I am pretty sure NOT on PRs, which is my confusion since you disabled the Azure pipeline for those.
Additionally you set this to run on Release publish (which confuses me a bit why, but is fine)